Base.rte/Devices/Tools/Scanner/ScannerHeavy.lua

```diff
@@ -19,17 +19,12 @@ function Update(self)
 			local actor = MovableMan:GetMOFromID(self.RootID);
 			if MovableMan:IsActor(actor) then
 				self.parent = ToActor(actor);
-				if self.HFlipped == false then
```

Comment:
I feel like these scanners can probably all share 1 script with another one customizing ranges and junk. Maybe I'm wrong though, I haven't looked into it, but on a cursory glance they look basically identical.